LambdaEngine/Source/Application/Win32/Win32Application.cpp

```diff
@@ -222,7 +222,7 @@ namespace LambdaEngine
 		GetProcessTimes(self, &ftime, &ftime, &fsys, &fuser);
 		memcpy(&sys, &fsys, sizeof(FILETIME));
 		memcpy(&user, &fuser, sizeof(FILETIME));
-		float64 percent = (sys.QuadPart - lastSysCPU.QuadPart) + (user.QuadPart - lastUserCPU.QuadPart);
+		float64 percent = float64((sys.QuadPart - lastSysCPU.QuadPart) + (user.QuadPart - lastUserCPU.QuadPart));
 		percent /= (now.QuadPart - lastCPU.QuadPart);
 		percent /= numProcessors;
```

Comment:
I might be off here, but cant you save the last Div operation -> percent /= (now.QuadPart - lastCPU.QuadPart) * numProcessors; ?

Reply:
I wrote that piece of code. And you are right it is more optimal to combine it to only one division. If Theo want to fix it it's up to him, I do that as a minor commit in another branch otherwise.

Reply:
Done